scripts/creation-portal-migration/logger.js

```diff
@@ -0,0 +1,36 @@
+var path = require("path");
+var fs = require("fs");
+var Logger = require("bunyan");
+
+const date =
+  new Date().getDate() +
+  "-" +
+  (new Date().getMonth() + 1) +
```

Comment:
@jayesh9747 something like below we can do
const currentDate = new Date();
const formattedDate = `${currentDate.getDate()}-${currentDate.getMonth() + 1}-${currentDate.getFullYear()}`;